someip2val/docker/vsomeip/examples/subscribe-sample.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (C) 2014-2017 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
same as above.

Also in any case we need to add Eclipse KUKSA (c) above to not give the impression BMW is responsible for whatever bad things we might have done to their code